test/tests.bats

```diff
@@ -660,7 +661,13 @@ EOF
   assert_success
 }
 
-@test "checking accounts: listmailuser" {
+@test "checking accounts: listmailuser (quotas disabled)" {
+  run docker exec mail /bin/sh -c "ENABLE_QUOTAS=0 listmailuser | head -n 1"
+  assert_success
+  assert_output '* user1@localhost.localdomain'
+}
```

Comment:
I was expecting one of these to fail, and sure enough the tests show that happened :stuck_out_tongue: :

```
not ok 306 checking accounts: listmailuser (quotas disabled) in 964ms
# (from function `assert_output' in file test/test_helper/bats-assert/src/assert.bash, line 239,
#  in test file test/tests.bats, line 667)
#   `assert_output '* user1@localhost.localdomain'' failed
# 
# -- output differs --
# expected : * user1@localhost.localdomain
# actual   : * user1@localhost.localdomain ( 12K / ~ ) [0%]
# --
# 
```

<details>
<summary>Stuff you already know</summary>

You're calling `sh -c` so `/root/.bashrc` is not loaded as expected. You originally added this ENV here to provide an explicit value and had conditional logic in `listmailuser` to use a default if it wasn't set explicitly.

Now with the `/etc/dms-settings` approach, you're dumping a copy of the `/root/.bashrc` contents for the vars but without the export (_not sure why you've opted for this difference vs sourcing the `/etc/dms-settings` and using `export` there_).

This overrides the ENV you provide always now with whatever `/etc/dms-settings` has, which I think was your intention.

</details>

Just need to tweak the test?

Reply:
I am not sure right now, why the latest test fail, but the test before didn't. Will investigate later.

Reply:
> why the latest test fail, but the test before didn't.

Expand the "Stuff you already know" section in my previous comment.

I thought that was pretty explanatory for why it failed now and didn't before.

Reply:
Thanks. I was in hurry this morning and didn't see your explanation. ATM I've no idea how to fix the test in an elegant way. Do you?
Otherwise, I think I'll remove the test.

Reply:
It wouldn't be an issue with the original approach I thought was going to be taken, with the conditional check handling or via an entrypoint :stuck_out_tongue: 

- You could probably just copy/paste [this small test file](https://github.com/docker-mailserver/docker-mailserver/blob/master/test/mail_with_mdbox.bats) and adjust it for the two quota tests. Each can just spin up their own container.
- You can comment the failing one out but keep it there with a TODO to fix it instead of outright removing it.
- You can sed the `/etc/dms-settings` file to set the ENV before calling the utility script.

That last suggestion is probably the way to go? I know the current approach won't work with `docker run ... setup ...` but that's not really an issue, users should run additional commands via `docker exec` instead.

Reply:
I think I fixed the test with [`b3c6af9` (#2264)](https://github.com/docker-mailserver/docker-mailserver/pull/2264/commits/b3c6af90f6b2369bfd8b12a0819f00577656f802). This is similar to your `sed` proposal.

There should be no problem when no container is running, as the source error is silent now.